src/test/data/script_tests.json

```diff
@@ -1856,7 +1754,7 @@
     "0x05 0x004c0245aa",
     "HASH160 0x14 0xd3ec673296c7fd7e1a9e53bfc36f414de303e905 EQUAL",
     "CLEANSTACK,P2SH",
-    "CLEANSTACK",
+    "MINIMALDATA",
```

Comment:
For context; the spirit of this check is to validate an invalid witness program; now the error is simply a bit earlier.